<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Помощник мастера настольной ролевой игры</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>741</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Настройка</string>
     </attribute>
     <widget class="QGroupBox" name="config_buttons">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>411</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="param_work">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>201</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Работа с параметрами персонажей</string>
       </property>
      </widget>
      <widget class="QPushButton" name="init_config">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>201</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Настройка инициативы</string>
       </property>
      </widget>
      <widget class="QPushButton" name="effects_work">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>210</y>
         <width>201</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Работа с основными эффектами</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="param_work_box">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>401</width>
        <height>411</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Выберите параметр</string>
       </property>
      </widget>
      <widget class="QListWidget" name="param_list">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>256</width>
         <height>121</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Параметр 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Параметр 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Параметр 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Параметр 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Параметр 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Параметр 6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Параметр 7</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="edit_param">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>210</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Редактировать</string>
       </property>
      </widget>
      <widget class="QPushButton" name="delete_param">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>210</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
      <widget class="QPushButton" name="create_param">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>260</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Создать параметр</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="init_config_box">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>320</y>
        <width>191</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="static_init_radio">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Статическая</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="dynamic_init_radio">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Динамическая</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Инициатива:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>150</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Выберите вляиющий на инициативу параметр:</string>
       </property>
      </widget>
      <widget class="QListWidget" name="param_list_init">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>180</y>
         <width>256</width>
         <height>131</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Параметр 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Параметр 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Параметр 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Параметр 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Параметр 5</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="ok_init_config">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>330</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cancel_init_config">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>330</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="effect_work_box">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>250</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Выберите эффект</string>
       </property>
      </widget>
      <widget class="QListWidget" name="effect_list">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>256</width>
         <height>121</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Эффект 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Эффект 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Эффект 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Эффект 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Эффект 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Эффект 6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Эффект 7</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="edit_effect">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>210</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Редактировать</string>
       </property>
      </widget>
      <widget class="QPushButton" name="delete_effect">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>210</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
      <widget class="QPushButton" name="create_effect">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>260</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Создать эффект</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
